2-mandatory/6-budgets.js

```diff
@@ -10,8 +10,27 @@ getBudgets([
 Should give return the answer of 62600.
 **/
 
+const getBudgets = (peopleArray) => {
+  return Object.values(peopleArray).reduce((t, { budget }) => t + budget, 0);
```

Comment:
calling Object.values(peopleArray) returns peopleArray. So why need to use an extra function? Great you know how to destructuring syntax an object.